src/decaylanguage/dec/decparser.py

```diff
@@ -223,25 +220,22 @@ def addline(self, line):
         if killindex is None:
             print(spl)
             raise Exception("No decay model specified: %s" % line)
-        #        print spl[killindex:]
         decay = Decay()
         decay.bf = bf
-        #        print self.current_decay_top.decay_of, 'to',
         for part in spl[1:killindex]:
-            #            print part,
```

Comment:
This is git, no commented out code please, unless there's a really good reason. Can always pull it from history.

It might be a great idea to add `logging` support, though, so you could enable the logger and see DEBUG or TRACE output from things like this. :)

Reply:
In fact this file is to be removed as it was what we got from Peter Onyisi in the old times. It is not quite working and ... But I will act on that soon.